pkg/providers/ovirt/templates/template-provider.go

```diff
@@ -0,0 +1,89 @@
+package templates
+
+import (
+	"fmt"
+	"strings"
+
+	templatev1 "github.com/openshift/api/template/v1"
+	v1 "github.com/openshift/api/template/v1"
+	tempclient "github.com/openshift/client-go/template/clientset/versioned/typed/template/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// TemplateOsLabel is a label that specifies the OS of the template
+	TemplateOsLabel = "os.template.kubevirt.io/%s"
+
+	// TemplateWorkloadLabel is a label that specifies the workload of the template
+	TemplateWorkloadLabel = "workload.template.kubevirt.io/%s"
+
+	// TemplateFlavorLabel is a label that specifies the flavor of the template
+	TemplateFlavorLabel = "flavor.template.kubevirt.io/%s"
+
+	processingURI = "processedTemplates"
+	nameParameter = "NAME"
+	otherValue    = "other"
+)
+
+// Templates is responsible for finding templates
+type Templates struct {
```

Comment:
This one also looks to be something that can be pulled out of ovirt provider scope.

Reply:
:+1: